cmd/dcrctl/config.go

```diff
@@ -178,13 +178,20 @@ func loadConfig() (*config, []string, error) {
 		RPCCert:    defaultRPCCertFile,
 	}
 
+	// Create the home directory if it doesn't already exist.
+	err := os.MkdirAll(btcdHomeDir, 0700)
```

Comment:
eesh I went back to test why I missed this and it was because I tested btcd instead of dcrd.  Good catch and thanks.